README.md

```diff
@@ -46,6 +46,16 @@ This runs the installed linters on the given repository. Linters need to be conf
 
 This runs unit tests on the given repository. The extension will try to guess the unit test framework to run based on project type, but it can also be configured in the configuration files.
 
+To use the status command, simply run:
+
+```sh
+gh dxp status
+```
+
+This command will display information about the current branch, uncommitted changes, and any changes that are committed but not yet pushed to the remote repository. It's a handy way to ensure that your work is organized and ready for review.
+
+Remember, staying informed about the status of your work helps streamline the development process and facilitates collaboration with your team.
```

Comment:
I think we should aim to keep the documentation as concise as possible. I suggest removing _"It's a handy way..." and "Remember, staying..."_